### PR TITLE
Update CircleCI image

### PR DESCRIPTION
**Pull request by pbackus**
See https://circleci.com/docs/next-gen-migration-guide/

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ jobs:
   build:
     working_directory: ~/dlang.org
     docker:
-      - image: circleci/node:4.8.2
+      - image: cimg/node:lts
```
Comment (ibuclaw) [inline .circleci/config.yml:6]: I'm pretty sure we're not using `node` anywhere in this repository.

**Comment by ibuclaw** [comment]
I'd prefer for this to target stable first, then merge from stable->master.  I was due to trigger a merge any day now, just as soon as @WalterBright answers his emails and gives me access to ftp.digitalmars.com. :-)

File: .circleci/config.yml (hunk 1 above)
Comment (pbackus) [reply .circleci/config.yml:6]: I'll try it with `cimg/base`.

**Comment by pbackus** [comment]
Rebased to stable.

**Review by thewilsonator — APPROVED** [review]

**Pull request merged**